src/SonarScanner.MSBuild.Shim/ScannerEngineInput.cs

```diff
@@ -84,11 +84,11 @@ public void AddVsTestReportPaths(string[] paths) =>
     public void AddVsXmlCoverageReportPaths(string[] paths) =>
         Add(SonarProperties.VsCoverageXmlReportsPaths, paths);
 
-    public void AddGlobalSettings(AnalysisProperties properties)
+    public void AddAllUserSettings(IAnalysisPropertyProvider properties)
```

Comment:
The `All` is not needed here
```suggestion
    public void AddUserSettings(IAnalysisPropertyProvider properties)
```